element_array_ephys/readers/kilosort_triggering.py

```diff
@@ -347,7 +347,7 @@ def __init__(self, npx_input_dir: str, ks_output_dir: str,
         self._json_directory = self._ks_output_dir / 'json_configs'
         self._json_directory.mkdir(parents=True, exist_ok=True)
 
-        self._median_subtraction_finished = False
+        self._median_subtraction_status = {}
```

Comment:
Reviewing the docstrings across readers, they appear to be in Sphinx format. They would need to be in Google format for proper [docs rendering](https://datajoint.com/docs/elements/element-array-ephys/0.2/api/element_array_ephys/readers/kilosort/) (vs. [dlc API](https://datajoint.com/docs/elements/element-deeplabcut/0.2/api/element_deeplabcut/readers/dlc_reader/))

Reply:
Agreed, will rework docs into Google format in the next PR